<!-- -->
<launch>

  <!-- send table.xml to param server -->
  <arg name="parent_frame" default="map" />
  <arg name="child_frame" default="gps" />
  <!-- <arg name="child_frame" default="velodyne" /> -->
  <arg name="points_topic" default="points_raw" />
  <arg name="scan_num" default="100" />
  <arg name="output_dir" default="/tmp/" />
  
  <node pkg="lidar_localizer" type="tf_mapping" name="tf_mapping" output="screen">
    <param name="parent_frame" value="$(arg parent_frame)" />
    <param name="child_frame" value="$(arg child_frame)" />
    <param name="points_topic" value="$(arg points_topic)" />
    <param name="scan_num" value="$(arg scan_num)" />
    <param name="output_dir" value="$(arg output_dir)" />
  </node>
  
</launch>
